src/components/common/end-test-link/end-test-link.ts

```diff
@@ -28,6 +33,11 @@ export class EndTestLinkComponent {
 
   onTerminate = () => {
     this.terminateTestModal.dismiss();
-    this.navController.push(CAT_B.DEBRIEF_PAGE);
+    if (this.category === TestCategory.BE) {
+      this.navController.push(CAT_BE.DEBRIEF_PAGE);
+    } else {
+      this.navController.push(CAT_B.DEBRIEF_PAGE);
```

Comment:
not a massive fan of defaulting it to CAT B as it's something we will probaly not easily notice in future categories. Could we do nothing or redriect to the error page.

Reply:
Changed to a switch statement.  It will do nothing if not a recognised category.